Mena_4pm_ProjectPortfolio_Team12.c

```diff
@@ -96,15 +104,80 @@ void DisplayBoard(int board[][7]) {
         printf("\n\n\n\n\n");
     }
 }
-int CheckNorth();
-int CheckEast();
-int CheckSouth();
-int CheckWest();
+int CheckNorth(int row, int col, int board[][7]) {
+    int count = 1;
+    int i = row;
+    if (i == 0) {
+        return 1;
+    }
+    while (board[i][col] == board[i-1][col]) {
+        count++;
+        i--;
+        if (i == 0) {
+            break;
+        }
+    }
+    return count;
+}
+int CheckEast(int row, int col, int board[][7]) {
+    int count = 1;
+    int i = col;
+    if (i == 6) {
+        return 1;
+    }
+    while (board[row][i] == board[row][i+1]) {
+        count++;
+        i++;
+        if (i == 6) {
+            break;
+        }
+    }
+    return count;
+}
+int CheckSouth(int row, int col, int board[][7]) {
+    int count = 1;
+    int i = row;
+    if (i  == 5) {
+        return 1;
+    }
+    while (board[i][col] == board[i+1][col]) {
+        count++;
+        i++;
+        if (i == 5) {
+            break;
+        }
+    }
+    return count;
+}
+int CheckWest(int row, int col, int board[][7]) {
+    int count = 1;
+    int i = col;
+    if (i == 0) {
+        return 1;
+    }
+    while (board[row][i] == board[row][i-1]) {
+        count++;
+        i--;
+        if (i == 0) {
+            break;
+        }
+    }
+    return count;
+}
 int CheckNorthEast();
 int CheckSouthEast();
 int CheckSouthWest();
 int CheckNorthWest();
 int CheckWinner();
+void TesterFunction(int board[][7]) {
+    //check the return value of north west east and south for a given i and j location
+    int row, colummn;
+    printf("Enter row and col: ");
+    scanf("%d%d", &row, &colummn);
+    printf("At row = %d and colummn = %d: \n", row, colummn);
+    printf("East is %d\nWest is %d\nNorth is %d\nSouth is %d\n", CheckEast(row, colummn, board), CheckWest(row, colummn, board), CheckNorth(row, colummn, board), CheckSouth(row, colummn, board));
+
+}
```

Comment:
This is the tester function use this in order to test new functions when done testing a scenario i recommend commenting it out instead of deleting it.